ui/k-toolbar/main.kytos

```diff
@@ -78,11 +79,12 @@
         </k-accordion-item>
 
         <span v-for="constraint in ['primary_constraints', 'secondary_constraints']">
-            <k-accordion-item :title="constraint_titles[constraint]"
-             v-if="false">
+            <k-accordion-item :title="constraint_titles[constraint]" :checked="false">
```

Comment:
@rmotitsuki, it looked great collapsed, but when using the component, when I was filling out the form values for instance spf max path cost it was constantly collapsing again, looks like there's something else bind reactively to `:checked` (maybe it'll need to be fixed on `ui` repo), its usability in its current state can get a bit annoying from an user perspective. Maybe we can leave this the collapsed part for a subsequent PR, and leave here without collapsing it?

Reply:
@rmotitsuki your commmit on https://github.com/kytos-ng/mef_eline/pull/337/commits/306b2dfa66a0ef6e5b6c7ae5838000351ebda1fd fixed the issue, nicely done. Let's close this thread.